src/stories/components/ChartTooltip/ChartTooltip.story.tsx

```diff
@@ -73,7 +73,7 @@ const lineData = [
 const LineTooltipStory: StoryFn<typeof ChartTooltip> = (args): ReactElement => {
 	const chartProps = useChartProps({ data: lineData, width: 600 });
 	return (
-		<Chart {...chartProps}>
+		<Chart {...chartProps} animations={false}>
```

Comment:
nit: let's be careful we don't let these slip into the storybook unless necessary